lib/matplotlib/tests/test_scale.py

```diff
@@ -55,6 +55,21 @@ def test_symlog_mask_nan():
     assert type(out) == type(x)
 
 
+def test_symlog_linthresh():
+    fig, ax = plt.subplots()
+
+    np.random.seed(19680801)
+    x = np.random.random(100)
+    y = np.random.random(100)
+
```

Comment:
```suggestion
    np.random.seed(19680801)
    x = np.random.random(100)
    y = np.random.random(100)

    fig, ax = plt.subplots()
```
It's slightly better to move the figure creation down to have all the plotting logic together.

Reply:
Definitely, makes the semantics more intuitive. Thanks @timhoffm